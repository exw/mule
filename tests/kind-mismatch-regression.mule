# This test currently fails during kind inference. It shouldn't
# type check, but it *is* well-kinded.
4 : (e : { type t a }) -> e.t int
