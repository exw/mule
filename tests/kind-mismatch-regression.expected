
Desugared:

	((:
	  ((e : (exist $0 (record (types ((t $0))) (values ())))) -> ((. e t) int)))
	 (Integer 4))
Type error: permission error during raise
